packages/components/src/Actions/Action/Action.component.js

```diff
@@ -58,11 +58,7 @@ function getActionComponent({ displayMode, getComponent }) {
  * @param {ActionProps}
  */
 function Action({ displayMode, getComponent, ...props }) {
-	const ActionComponent = getActionComponent({
-		displayMode,
-		getComponent,
-		...props,
-	});
+	const ActionComponent = getActionComponent({ displayMode, getComponent });
```

Comment:
No need to spread `props`, `getActionComponent` only reads `displayMode` and `getComponent`